### PR TITLE
Super tiny typo fixes

### PR DESCRIPTION
**Pull request by cpavanrun**
Just some typo's I found while reading the code base. It's a nice pull-request tryout before I start doing other major things. 

**Note**: Travis CI is failing because of an earlier commit. Something completely totally unrelated.

### CODE REVIEW

**Comment by cpavanrun** [comment]
Reflecting I made a mistake and the pull-request should be repeated at a different point in time.

**Comment by ewels** [comment]
Hi @cpavanrun,

Thanks for this! Why did you close the PR? Timing looked fine to me :)

Phil

File: multiqc/modules/fastqc/fastqc.py [hunk 1]
```diff
@@ -317,7 +317,7 @@ def per_seq_quality_plot (self):
         pconfig = {
             'id': 'fastqc_per_sequence_quality_scores_plot',
             'title': 'FastQC: Per Sequence Quality Scores',
-            'ylab': 'Count',
+            'ylab': 'Percentage',
```
Comment (ewels) [inline multiqc/modules/fastqc/fastqc.py]: Is this percentages? In the plot I'm looking at it goes up to 300 million so I'm pretty sure it's counts for me. What does the FastQC report say?

File: multiqc_config_example.yaml [hunk 2]
```diff
@@ -61,7 +61,7 @@ fn_ignore_dirs:
 fn_ignore_paths:
     - '*/path/to/*_files/'
 
-# Ignore files larger than this when searcing for logs (bytes)
+# Ignore files larger than this when searching for logs (bytes)
```
Comment (ewels) [inline multiqc_config_example.yaml]: Good spot 👍

**Comment by cpavanrun** [comment]
Hi Phil,

Closed it because one of the fixes I did was incorrect and at the wrong place =/. Was a bit too eager to PR. Planning on re-doing it today.

Cheers,

Chris

--
Chris van Run, MSc | Bio-Informatician | Princess Máxima Centre for pediatric oncology

**Comment by ewels** [comment]
Ok cool - for future reference, you don't need to close and reopen pull requests in this case. Just keep pushing commits and the pull request will update accordingly.

**Comment by cpavanrun** [comment]
Thank you Phil. Will take that into account in the near future! 

I wasn't _sure_ because the new commits didn't show when the PR is closed.

**Pull request closed without merging**